readme-tw-gtasks.md

```diff
@@ -0,0 +1,87 @@
+# [Taskwarrior](https://taskwarrior.org/) ⬄ [Google Tasks](https://support.google.com/tasks/answer/7675772?hl=en&co=GENIE.Platform%3DDesktop)
```

Comment:
Could you remove the part after `?`, i.e., keep just the `https://support.google.com/tasks/answer/7675772`

Reply:
done in latest revision